### ISSUE TITLE
For seeding formats

### ISSUE THREAD

**Issue by dengpeiyao0515** [Riscure/optee_fuzzer#9]
Hi there,

I am wondering if the hex format for collected system call sequences was defined by you?
![0](https://user-images.githubusercontent.com/74812838/111966070-72c36b00-8b31-11eb-9fc5-4208e85062cb.png)
If so, would it be possible for you to give some documentation links to understand the format?

**Comment by MartijnB** [comment]
Yes, this is my own format. There is no documentation about it. The data structures can be found here: https://github.com/Riscure/optee_fuzzer/blob/master/shared/include/afl-tee.h#L45-L102

It is basically an array of SYSCALL_INVOKEs followed by arbitrary data that can be referenced by an ARG_DATA_* argument. The different argument types allow providing values, pointers to buffers, a value by dereferencing a pointer, etc. This way you can construct pretty much any chain of function calls.
